its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/IPythonTest.java

```diff
@@ -84,50 +85,52 @@ public static void stop() {
   }
 
   @Test
-  public void should_raise_issues() throws IOException {
-    ClientInputFile inputFile = prepareInputFile("foo.ipynb",
-      "def fooBar():\n"
-        + "  `1` \n"
-        + "  `1` #NOSONAR\n",
-      false);
-
-    List<Issue> issues = new ArrayList<>();
-    StandaloneAnalysisConfiguration configuration =
-      StandaloneAnalysisConfiguration.builder()
-        .setBaseDir(baseDir.toPath())
-        .addInputFile(inputFile)
-        .setModuleKey("myModule")
-        .build();
-
-    Map<ClientLogOutput.Level, List<String>> logsByLevel = new HashMap<>();
-    ClientLogOutput logOutput = (s, level) -> {
-      List<String> logs = logsByLevel.getOrDefault(level, new ArrayList<>());
-      logs.add(s);
-      logsByLevel.putIfAbsent(level, logs);
-    };
-    ClientModuleFileSystem clientFileSystem = new ClientModuleFileSystem() {
+  public void shouldRaiseIssues() throws IOException {
```

Comment:
It seems the `throws IOException` can be removed.

Reply:
done